modules/Microsoft.AAD/DomainServices/.test/common/dependencies.bicep

```diff
@@ -15,21 +15,22 @@ param certDeploymentScriptName string
 
 var certPWSecretName = 'pfxCertificatePassword'
 var certSecretName = 'pfxBase64Certificate'
+var addressPrefix = '10.0.0.0/16'
```

Comment:
I'd go with a /26 max, 64 IPs is already enough for a test

Reply:
But does that matter? Should we use a different cidr for each test - i.e. the one that is needed? Personally I'd prefer to keep the address spaces somewhat aligned as it makes seach & replace easier (if need be). 

Currently, aside from tests with multiple subnets, we either use /24 or /16.

Reply:
It doesn't matter to specify for each test a different one or the one needed. I'd at least avoid to use a /16. I think /26 would be enough for all tests

Reply:
I'd suggest to keep it as /16. There is no harm to have a bigger one. Azure resources subnet recommendations are not standardised. Example: App GW recommendation is a /24, Vwan Hub /24, azure Vmware /22. ASE I think is also /24.

Reply:
Accepted 😜